### PR TITLE
 adds cypress dep and fixes node modules git ignore

### PR DESCRIPTION
**Pull request by ManuelSaleta**

### CODE REVIEW

File: .gitignore
Comment (GitNey) [inline .gitignore:1]: Noice. I actually added one of these in the root though (In PR #2). I think that's where it oughta be the `.gitignore`. In `/.gitignore` not `src/.gitignore`. 


Screen Shot 2023-09-13 at 11 16 01 AM

**Review by GitNey — CHANGES_REQUESTED** [review]
One small request sire.

File: .gitignore
Comment (GitNey) [reply .gitignore:1]: Having `node_modules/` in a `src/.gitignore` presumes that directory is in the same level. `node_modules/` is at the root level.

Comment (ManuelSaleta) [reply .gitignore:1]: yes its actually moved in the PR. 
image

**Review by GitNey — APPROVED** [review]

**Pull request merged**